packages/proto/package.json

```diff
@@ -15,6 +15,7 @@
   "author": "Roey Berman <roey@temporal.io>",
   "license": "MIT",
   "dependencies": {
+    "@temporalio/common": "file:../common",
```

Comment:
This is a circular dependency

Reply:
The dep is for: 

```
const { patchProtobufRoot } = require('@temporalio/common/lib/converter/patch-protobuf-root');
```

Shall I move the `patch-protobuf-root.ts` file to `packages/protos/src/`?

Reply:
Then `packages/common/src/converter/patch-protobuf-root.ts` would have:

```ts
export { patchProtobufRoot } from '@temporalio/proto/lib/patch-protobuf-root'
```

Reply:
Sure, we might just need to re-export this function in common.